src/main/java/frc/robot/subsystems/ControlPanel.java

```diff
@@ -0,0 +1,94 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj.util.Color;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatch;
+import edu.wpi.first.wpilibj.I2C;
+
+public class ControlPanel extends SubsystemBase {
+
+  private final WPI_TalonSRX m_controlPanelMotor = new WPI_TalonSRX(Constants.ControlPanelConstants.kSpinnerMotorPort);
+  private final ColorMatch m_colorMatch = new ColorMatch();
+  private final I2C.Port i2cPort = I2C.Port.kOnboard;
+  private final ColorSensorV3 m_colorSensor = new ColorSensorV3(i2cPort);
+
+     String color;
+
+  /**
+   * Creates a new ControlPanel.
+   */
+  public ControlPanel() {
+
+    m_controlPanelMotor.configFactoryDefault();
+    addChild("Control Panel Motor", m_controlPanelMotor);  
+
+  }
+
+  // starts the control panel motors
+  public void startMotors(double speed) {
+
+    m_controlPanelMotor.set(speed);
+
+  }
```

Comment:
There's only one motor, so a more descriptive method might be spinWheel(double speed).